### DISCUSSION TITLE
my cpu is ussage is high ram and both storage allso but gpu is at 1-5 %

### DISCUSSION THREAD

**Discussion by Ok12340** [lllyasviel/Fooocus#3532] [Q&A]
my cpu is ussage is high ram and both storage allso but gpu is at 1-5 % . and creating an image works but takes very long
![Screenshot (13)](https://github.com/user-attachments/assets/c7417c5b-7ab7-47dd-9f20-34e9f9a651b0)

**Comment by lgwjames** [comment]
please post specs off set up

**Reply by Ok12340** [reply]
No i dont wanted to do that but i wanted to show that it doesnt use my gpu but i want it to do so

**Comment by Raito00** [comment]
Yes, if PC have two GPUs how to set use specific GPU?
1. AMD Radeon GPU
2. NVIDIA RTX 3050 TI

**Reply by Rogala** [reply]
Edit run.bat [--gpu-device-id DEVICE_ID](https://github.com/lllyasviel/Fooocus#all-cmd-flags)
